src/main.ts

```diff
@@ -14,12 +14,12 @@ const name = core.getInput('name', { required: true });
 const extraPullNames = core.getInput('extraPullNames');
 const signingKey = core.getInput('signingKey');
 const authToken = core.getInput('authToken')
-const skipPush = core.getInput('skipPush');
+const skipPush = core.getBooleanInput('skipPush', { required: true });
```

Comment:
Do we still need `required: true` if there's a default?

Reply:
Good call; I think I misread the description here:

> If required set to be **false**, the input should have a default value in action.yml.

I read it as `true` the first time 😅 

https://github.com/actions/toolkit/tree/main/packages/core#inputsoutputs